File: src/Paiza/python/D004.py
num = int(input())
s = []
s.append("Hello ")

for i in range(0,num):
    s2 = input()
    if i == num-1:
        s.append(s2+".")
    else:
        s.append(s2+",")

s = ''.join(s)
print(s)

File: src/Paiza/D004.kt
package Paiza

import java.lang.StringBuilder

fun main() {
    val num = readLine()!!.toInt()
    val str = StringBuilder()
    str.append("Hello ")
    for (i in 0 until num){
        val str2 = readLine()!!
        if (i == num-1){
            str.append("$str2,")
        }else{
            str.append("$str2.")
        }
    }

    println(str)


}